test/files/neg/t11631.scala

```diff
@@ -0,0 +1,22 @@
+
+// scalac: -Xsource:3 -Werror -Wunused
+
+import annotation.*
+
+class C {
+  def n: Int = 42
+  def f(@unused i: Int) = n + 1
+  def g(@unused i: Int) = i + 1   // unused unused
```

Comment:
unused unused doesn't seem to be reported?

Reply:
I wasn't able to workaround the spurious report for `ChromeTrace.EventType`, so I put in a version trip switch.

That is also why I fixed the ordering of `ScalaVersion` so that `2.13.11-development < 2.13.11`.

Reply:
Actually, the problem of the unused diagnostic not seeing trees of constant values made me want to address the few well-known limitations with the unused checks for 2.13.11, depending perhaps on when it is scheduled.